plugins/woocommerce/src/Utilities/OrderUtil.php

```diff
@@ -257,4 +257,14 @@ public static function remove_status_prefix( string $status ): string {
 
 		return $status;
 	}
+
+	/**
+	 * Checks if the new full refund data is used.
+	 *
+	 * @return bool
+	 */
+	public static function uses_new_full_refund_data() {
+		$db_version = get_option( 'woocommerce_db_version', null );
+		return version_compare( $db_version, '10.2.0', '>=' ) && 'no' === get_option( 'woocommerce_analytics_uses_old_full_refund_data', 'no' );
```

Comment:
Just found that we should check if $db_version is null before comparing it. `version_compare` doesn't accept null as an argument. In PHP +8, it will throw a deprecation warning: `Deprecated: version_compare(): Passing null to parameter #1 ($version1) of type string is deprecated in `